src/index.css

```diff
@@ -14,6 +18,9 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
+  position: relative;
+  background-color: rgba(252, 81, 147, 255);
 }
 
 code {
```

Comment:
- [ ] Please use the correct font families in your design, `gill sans` as body and title font and `lato` as control font